docs/network-flow-visibility.md

```diff
@@ -0,0 +1,101 @@
+# ELK Flow Collector
+## Purpose
+Antrea supports sending IPFIX flow records through a flow exporter. The Elastic
+Stack (ELK Stack) works as the data collector, data storage and visualization tool
+for flow records and flow-related information. This doc provides guidelines for
+deploying Elastic Stack with support for Antrea-specific fields in a Kubenetes
+cluster.
+
+## About Elastic Stack
+[Elastic Stack](https://www.elastic.co) is a group of open source products to
+help collect, store, search, analyze and visualize data in real time. We will
+use Logstash, Elasticsearch and Kibana in Antrea flow visualization.
+[Logstash](https://www.elastic.co/logstash) works as data collector to
+centralize flow records. [Logstash Netflow codec plugin](https://www.elastic.co/guide/en/logstash/current/plugins-codecs-netflow.html)
+supports Netflow v5/v9/v10(IPFIX) protocols for flow data collection.
+The flow exporter feature in Antrea Agent uses the IPFIX (Netflow v10) protocol 
+to export flow records.
+
+Exported IPFIX flow records contain the following Antrea-specific fields along 
+with standard IANA fields.
+
+| IPFIX Information Element | Enterprise ID | Field ID | Type        |
+|---------------------------|---------------|----------|-------------|
+| sourcePodNamespace        | 55829         | 100      | string      |
+| sourcePodName             | 55829         | 101      | string      |
+| destinationPodNamespace   | 55829         | 102      | string      |
+| destinationPodName        | 55829         | 103      | string      |
+| sourceNodeName            | 55829         | 104      | string      |
+| destinationNodeName       | 55829         | 105      | string      |
+| destinationClusterIP      | 55829         | 106      | ipv4Address |
+| destinationServicePortName| 55829         | 108      | string      |
+
+[Elasticsearch](https://www.elastic.co/elasticsearch/), as a RESTful search
+engine, supports storing, searching and indexing records received. 
+[Kibana](https://www.elastic.co/kibana/) is mainly for data visualization and
+exploration.
+
+## Deployment Steps
+To create all the necessary resources in the `elk-flow-collector` namespace
+and get everything up-and-running, run:
+```shell
```

Comment:
can you add an empty line before and after the code block? I think Cody reported issues with netlify (website) without them.

Reply:
Done.